### PR TITLE
[translation] Nederlands/Dutch (nl-NL)

### PR DESCRIPTION
**Pull request by mroeling**
Translations based on en-EN language file, Franz 5.0.0-Beta10.

### CODE REVIEW

File: src/i18n/locales/nl-NL.json [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  "settings.account.headlineProfile": "Update profiel",
+  "settings.account.headlineAccount": "Account informatie",
+  "settings.account.headlinePassword": "Wijzig wachtwoord",
+  "settings.account.successInfo": "Je wijzigingen zijn opgeslagen",
+  "settings.account.buttonSave": "Update profiel",
+  "settings.account.mining.thankyou": "Bedankt voor je ondersteuning van Franz met je rekenkracht.",
+  "settings.account.mining.active": "Op dit moment voer je {hashes} berekingen per seconde uit.",
+  "settings.account.mining.moreInformation": "Meer informatie",
+  "settings.account.mining.cancel": "Stop mining",
+  "settings.navigation.availableServices": "Beschikbare modules",
```
Comment (gdm-1718-robbhaeg) [inline src/i18n/locales/nl-NL.json]: it's more fluent to use services instead of modules

File: src/i18n/locales/nl-NL.json [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  "settings.navigation.yourServices": "Jouw modules",
```
Comment (gdm-1718-robbhaeg) [inline src/i18n/locales/nl-NL.json]: it's more fluent to use services instead of modules

File: src/i18n/locales/nl-NL.json [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  "settings.navigation.account": "Account",
+  "settings.navigation.settings": "Instellingen",
+  "settings.navigation.logout": "Uitloggen",
+  "settings.recipes.headline": "Beschikbare modules",
```
Comment (gdm-1718-robbhaeg) [inline src/i18n/locales/nl-NL.json]: it's more fluent to use services instead of modules

File: src/i18n/locales/nl-NL.json [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+  "settings.recipes.mostPopular": "Meest populair",
+  "settings.recipes.all": "Alle modules",
```
Comment (gdm-1718-robbhaeg) [inline src/i18n/locales/nl-NL.json]: it's more fluent to use services instead of modules

File: src/i18n/locales/nl-NL.json [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+  "settings.recipes.dev": "Ontwikkeling",
+  "settings.recipes.nothingFound": "Geen module kwam overeen met je zoekopdracht.",
```
Comment (gdm-1718-robbhaeg) [inline src/i18n/locales/nl-NL.json]: it's more fluent to use service instead of module

**Comment by mroeling** [comment]
I did hesitate to use either. In dutch the word service has a different meaning than in English, less usefull as a plural word. That's why I choose to use modules instead.

**Comment by mroeling** [comment]
@gdm-1718-robbhaeg I have updated the file to reflect your comments.

File: src/i18n/languages.js [hunk 6]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
   'en-US': 'English',
+  'nl-NL': 'Nederlands',
```
Comment (adlk) [inline src/i18n/languages.js]: `nl` as a key would be sufficient. I will change this once it's merged.

**Pull request merged**